be/src/olap/tablet_schema_cache.h

```diff
@@ -32,19 +33,18 @@ class TabletSchemaCache {
 public:
     ~TabletSchemaCache();
 
-    static void create_global_schema_cache() {
-        DCHECK(_s_instance == nullptr);
-        static TabletSchemaCache instance;
-        _s_instance = &instance;
-        std::thread t(&TabletSchemaCache::_recycle, _s_instance);
+    static TabletSchemaCache* create_global_schema_cache() {
```

Comment:
We'd better add a seperate shutdown method for this class and add a start method to start the thread.